src/components/molecules/UserList/UserList.scss

```diff
@@ -7,6 +7,10 @@ $user-list-cell-size: 30px;
     display: flex;
     flex-direction: column;
     align-items: center;
+
+    &-text {
+      z-index: z(user-list-label);
```

Comment:
why do we need to increase z-index for the text?

Reply:
ref https://github.com/sparkletown/sparkle/pull/2265#discussion_r708476609